srcStatic/ConsoleModuleLoader.h

```diff
@@ -6,10 +6,14 @@
 class ConsoleModuleLoader {
 public:
 	ConsoleModuleLoader();
+	// Test Constructor
+	ConsoleModuleLoader(const std::string& testModuleDirectory);
```

Comment:
There's something about the comment that makes me want to remove it. I can understand why it's there though, and it probably should be there. Perhaps this relates to my earlier comments about dependency injection, and wondering if the constructors should be reworked such that a comment wouldn't be of value. Anyway, food for thought. No changes needed.

Reply:
Once someone refactors ConsoleModuleLoader to only have 1 constructor, won't be an issue anymore. In its current state, should not be used outside test environment.